.eslintignore

```diff
@@ -0,0 +1,13 @@
+# editorconfig.org
```

Comment:
The reference to jshint should be changed in this section https://github.com/telefonicaid/iotagent-ul/blob/master/docs/usermanual.md#coding-guidelines

(Maybe the same in the other associated PRs)

Reply:
Fixed - 97d31b3 b15e1b4 95d5b7e